packages/neutron-sdk/src/bindings/msg.rs

```diff
@@ -110,6 +110,9 @@ pub enum NeutronMsg {
         timeout_timestamp: u64,
         fee: IbcFee,
     },
+    /// SubmitAdminProposal sends a proposal to neutron's Admin module.
+    /// This type of messages can be only executed by Neutron DAO.
+    SubmitAdminProposal { admin_proposal: AdminProposal },
```

Comment:
I think it could be a good idea to hide the variant by some feature flag, lets say "dao", and hide other admin/dao specific types and functions as well

Reply:
that's an interesting but not really useful idea. I'd discuss whether we really need it and, if yes, made it as a separate task